api/src/main/java/ca/bc/gov/educ/api/graduation/service/ReportService.java

```diff
@@ -236,13 +237,22 @@ private void createAssessmentListForTranscript(List<StudentAssessment> studentAs
 				List<StudentCourse> newList= new ArrayList<>();
 				List<StudentCourse> provinciallyExaminable = studentCourseList.stream().filter(sc -> sc.getProvExamCourse().compareTo("Y")==0).collect(Collectors.toList());
 				if(!provinciallyExaminable.isEmpty()) {
-					provinciallyExaminable.sort(Comparator.comparing(StudentCourse::getCourseCode).thenComparing(StudentCourse::getCourseLevel));
+					if(program.contains("1950")) {
```

Comment:
Hi @sreepadkarekkad, some code duplication here and on line 250. Can you create a function that takes a List of StudentCourse objects and returns the sorted result? Thanks.